x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/utils.ts

```diff
@@ -171,8 +184,29 @@ export const singleBulkIndex = async ({
   logger.debug(`individual bulk process time took: ${time2 - time1} milliseconds`);
   logger.debug(`took property says bulk took: ${firstResult.took} milliseconds`);
   if (firstResult.errors) {
-    logger.error(`[-] bulkResponse had errors: ${JSON.stringify(firstResult.errors, null, 2)}`);
-    return false;
+    // go through the response status errors and see what
+    // types of errors they are, count them up, and log them.
+    const errorCountMap: { [key: string]: number } = {};
+    firstResult.items.forEach(item => {
+      if (item.create.error) {
+        const responseStatusKey = item.create.status.toString();
+        errorCountMap[responseStatusKey] = errorCountMap[responseStatusKey]
+          ? errorCountMap[responseStatusKey] + 1
+          : 1;
+      }
+    });
```

Comment:
Do you want a reduce for this so you return the errorCountMap instead of mutation?